<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteWindow</class>
 <widget class="QDialog" name="NoteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="txtNote">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>661</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblNoteDate">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>13</y>
     <width>691</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveNote">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>382</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNoteCancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinIm">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> Важность:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
